wolox-books/src/app/screens/BooksList/components/books-list/books-list.component.ts

```diff
@@ -1,23 +1,27 @@
 import { Component, OnInit } from '@angular/core';
 import { BookService } from 'app/services/book.service';
+import { FilterBooksPipe } from 'app/pipes/filter-books.pipe';
 import { Book } from 'app/models/book';
+import { Router } from '@angular/router';
+import * as Constants from 'app/utils/constants';
 
 @Component({
   selector: 'app-books-list',
   templateUrl: './books-list.component.html',
   styleUrls: ['./books-list.component.scss']
 })
-export class BooksListComponent {
+export class BooksListComponent implements OnInit {
 
   books: Book[] = [];
+  filterText = '';
+  openedModal = false;
+  routerLinks = Constants.routerLinks;
 
-  constructor(private bookService: BookService) { }
+  constructor(private bookService: BookService, private router: Router) { }
 
   ngOnInit(): void {
     this.bookService.getBooks().subscribe(
-      res => {
-        this.books = res['page'].map(book => ({ image: book.image_url, author: book.author, title: book.title }));
-      },
+      res => this.books = res['page'].map(book => book),
```

Comment:
I think, this map is unnecesary
```suggestion
      res => this.books = res.page,
```

Reply:
No, this doesn't work, it makes a compilation error, because as a callback, it doesn't know the inner properties of `res` so I have to call it as brackets notation. This was commented by @mcallegari10 and I answered him in this comment: https://github.com/wolox-training/jr-angular/pull/21#discussion_r427470189